account/connect.go

```diff
@@ -235,7 +235,7 @@ func (a *Service) openChannel(l lsp, force bool) error {
 	a.log.Info("openChannel started...")
 	_, err, _ := createChannelGroup.Do("createChannel", func() (interface{}, error) {
 		err := a.connectAndOpenChannel(l, force)
-		a.log.Info("openChannel finished, error = %v", err)
+		a.log.Infof("openChannel finished, error = %v", err)
```

Comment:
Can you please put all the small fixes in a separate commit? If you can put them in a separate PR, that would be even better.